pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -304,6 +304,11 @@ func (plugin *kubenetNetworkPlugin) Capabilities() utilsets.Int {
 // TODO: Don't pass the pod to this method, it only needs it for bandwidth
 // shaping and hostport management.
 func (plugin *kubenetNetworkPlugin) setup(namespace string, name string, id kubecontainer.ContainerID, pod *v1.Pod, annotations map[string]string) error {
+	// Disable DAD so we skip the kernel delay on bringing up new interfaces.
+	if err := plugin.disableContainerDAD(id); err != nil {
+		glog.V(4).Infof("Failed to disable DAD in container: %v", err)
```

Comment:
I don't have a strong opinion - but this seems to be important enough to be V(2). It's important setup-related and one-off msg.

Reply:
I thought about it for a bit, but it doesn't break anything or affect functionality in any way - it only costs an extra second of pod latency. But I don't have a strong opinion.